galaxy.yml

```diff
@@ -3,6 +3,7 @@ authors:
   - Ansible Network Community (ansible-network)
 dependencies:
   "ansible.netcommon": ">=8.1.0"
+  "juniper.device": ">=1.0.9"
```

Comment:
I don;t see  required changes in this release https://github.com/Juniper/ansible-junos-stdlib/tree/v1.0.9/ansible_collections/juniper/device